CMakeLists.txt

```diff
@@ -97,6 +98,13 @@ if (SSG_SVG_IN_XCCDF_ENABLED AND NOT OSCAP_SVG_SUPPORT_RESULT EQUAL 0)
     message(WARNING "Your version of OpenSCAP does not support having the SVG logo in the XCCDF, disabling SVG logo.")
 endif()
 
+if (SSG_JINJA2_CACHE_ENABLED)
+    file(MAKE_DIRECTORY "${CMAKE_BINARY_DIR}/jinja2_cache")
+    set(SSG_JINJA2_CACHE_DIR "${CMAKE_BINARY_DIR}/jinja2_cache")
```

Comment:
I'd make this configurable and default it to the build subdirectory if it is not set when `SSG_JINJA2_CACHE_ENABLED=true`. If I understand how they're implementing the [cache](http://jinja.pocoo.org/docs/2.10/api/#bytecode-cache) (and the [bucket](http://jinja.pocoo.org/docs/2.10/api/#jinja2.bccache.Bucket)), it is safe to use a global cache. That way, on the jenkins workers, we can use a shared global cache across all PRs, improving all build times.

(I'll mock these assumptions tomorrow to validate they hold).

Reply:
That's a pretty cool idea. Keep in mind though that jinja2 will never clean any bytecode, we would have to set-up some periodic cache purging to avoid it taking all disk space.

Reply:
I have changed the option to a PATH cmake variable. Now you can set it to any directory.